R/get_team_schedule.R

```diff
@@ -0,0 +1,77 @@
+#' Extract Team Schedule
+#'
+#' @param team_id Team ID from a link like: https://stats.ncaa.org/teams/561648
+#' @keywords schedule
+#' @export
+#' @examples get_team_schedule(525659)
+#'
+#'
+#'
+#'
+
+get_team_schedule <- function(team_id) {
+  library(tidyverse)
```

Comment:
for R packages, if you are introducing new packages you want to add them in the description. Don't want to load the libraries within a function. 

Drop these. 

A preferred method that I learnt recently is to do explictly call functions from the package like `dplyr::bind_rows` etc